pkg/api/types.go

```diff
@@ -921,6 +921,9 @@ type PodSpec struct {
 	// used must be specified.
 	// Optional: Default to false.
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// Run the container using the host ipc mode.
+	// Optional: Default to false.
+	HostIPC bool `json:"hostIPC,omitempty"`
```

Comment:
I agree with granular controls. We can add knobs as needed for now, but I predict we'll eventually want something more like a list, as with capabilities and resources.